source/settings.txt

```diff
@@ -1,5 +1,112 @@
 .. _vsce-settings:
 
-============================
-MongoDB for VS Code Settings
-============================
+===============
+|vsce| Settings
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Configure Settings
+------------------
+
+To configure |vsce| settings:
+
+.. include:: /includes/steps/configure-settings.rst
+
+Settings
+--------
+
+The following table lists the available settings for 
+|vsce|:
+
+.. list-table::
+   :widths: 25 50 25
+   :header-rows: 1
+   
+   * - Setting
+     - Description
+     - Default
+
+   * - :guilabel:`Shell`
+     - MongoDB shell to use. Choose either:
+       
+       - ``mongo`` (the legacy MongoDB shell), or
+
+       - ``mongosh`` (the new MongoDB shell).
+
+     - ``mongo``
+
+   * - :guilabel:`Show`
+     - If checked, shows the MongoDB view in the Visual Studio Code 
+       left navigation panel.
+     - Checked
+
+   * - :guilabel:`Max Number of Problems`
+     - The maximum number of problems |vsce| will generate 
+       in the Visual Studio Code :guilabel:`Problems` view.
+     - ``100``
+
+   * - :guilabel:`Trace: Server`
+     - MongoDB language server logging settings. 
+     
+       Edit this setting through the 
+       ``mongodbLanguageServer.trace.server`` property in your project's
+       ``settings.json`` file. 
+
+       Within the ``mongodbLanguageServer.trace.server`` object, you 
+       can specify:
+
+       - ``format``, and
+
+       - ``verbosity``
+       
+       Output logs are viewable by selecting 
+       :guilabel:`MongoDB Language Server` from the dropdown in the 
+       :guilabel:`Output` view, or with the 
+       `Language Server Protocol Inspector <https://github.com/Microsoft/language-server-protocol-inspector>`__.
+
+     - ``{``
+         ``"format": "json",``
+         ``"verbosity": "verbose"``
+       ``}``
+
+   * - :guilabel:`Confirm Run All`
+     - If checked, shows a confirmation message before running 
```

Comment:
[consideration] Throughout, I might consider replacing checked / unchecked with enabled / disabled. Enabled / disabled feels more like what I'm used to seeing in our docs, but I don't feel strongly about this.